mashina.Inc/src/main/java/com/epam/machine/Controller/userServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.machine.Controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class userServlet extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
+        try {
+            httpServletRequest.getRequestDispatcher("/userServlet.jsp").forward(httpServletRequest, httpServletResponse);
+        } catch (ServletException | IOException err) {
+            System.out.println("Something is wrong");
```

Comment:
u can also add e.getMessage() into log message

Reply:
And plz be more attentive in the naming of classes.
See code convention for more info

Reply:
And please rename all *.jsp files according to convention